x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -1401,6 +1430,16 @@ private EventFilterPolicy changeIndicesFilter(List<String> filtersList) {
             return l.stream().map(f -> f.isEmpty() ? "//" : f).collect(Collectors.toList());
         }
 
+        private boolean ignorePrivilegesPredicateTest(String action) {
+            Collection<String> privileges = null;
+            if (AuthorizationService.isIndexAction(action)) {
+                privileges = IndexPrivilege.findPrivilegesThatGrant(action);
+            } else if (ClusterPrivilegeResolver.isClusterAction(action)) {
+                privileges = ClusterPrivilegeResolver.findPrivilegesThatGrant(action, null, null);
```

Comment:
I think there will be problems with the `manage_own_api_key` privilege.
In general, I dislike passing `null`s like this, it looks brittle to me.
I think if we've got the option to pass along the `authentication` and `request` parameters, we should.

Reply:
Is the current proposed solution that for a cluster privilege filter with `manage_own_api_keys` to ignore all audits for token actions (ie work like `manage_api_key`)?
I don't think this is the best alternative. I'd rather not permit this privilege in an ignore policy. Just my opinion, take it from someone that is not a fan of named privileges for ignore policies.

Reply:
Changed to use pemission check instead + added targeted tests